drupal.make.yml

```diff
@@ -25,12 +25,16 @@ projects:
     version: ~
   features_extra:
     version: ~
+  features_override:
```

Comment:
Why this?  I can count the number of times i've ever needed to use features override and it worked on 1/2 finger.